.github/workflows/bf-frontend-tests.yml

```diff
@@ -3,10 +3,11 @@ name: (BF) Lint, Unit and Component tests
 on:
   pull_request:
     paths:
-    - 'frontend/benefit/**'
-    - 'frontend/shared/**'
-    - 'frontend/*'
-    - '.github/workflows/bf-frontend-tests.yml'
+      - "frontend/shared/**"
```

Comment:
This is probably the reason for linter problems, you divided this file to applicant and handler files but the original file is also here and might cause some strange side effecrts ... so you can delete this original file now.

Reply:
Also I didnt find the tests for the benefit-shared folder? So add that one too, you can copy `ks-shared-frontend-tests.yml` as template file.

Reply:
there are none yet, thats why i skipped it. Its just project template with shared types. later we have to move logic there

Reply:
removed now after merging develop, and added shared